sql/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -0,0 +1,91 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod`.telemetry_derived.experiment_enrollment_aggregates_live AS (
+  WITH live AS (
+    SELECT
+      event_object AS `type`,
+      event_string_value AS experiment,
+      `moz-fx-data-shared-prod`.udf.get_key(event_map_values, 'branch') AS branch,
+      TIMESTAMP_ADD(
+        TIMESTAMP_TRUNC(`timestamp`, HOUR),
+        INTERVAL (
+          CAST(
+            EXTRACT(
+              MINUTE
+              FROM
+                `timestamp`
+            ) / 5 AS int64
+          ) * 5
```

Comment:
casting a float to an integer will round, not truncate. I believe what we should actually do here is use the integer division function [DIV](https://cloud.google.com/bigquery/docs/reference/standard-sql/mathematical_functions#div)

```diff
-          CAST(
-            EXTRACT(
-              MINUTE
-              FROM
-                `timestamp`
-            ) / 5 AS int64
-          ) * 5
+          DIV(
+            EXTRACT(MINUTE FROM `timestamp`),
+            5
+          ) * 5
```

Reply:
This is what I did for error aggregates https://github.com/mozilla/bigquery-etl/blob/master/udf/round_timestamp_to_minute.sql

Reply:
filed #530 to fix that